java/code/src/com/redhat/rhn/frontend/xmlrpc/errata/ErrataHandler.java

```diff
@@ -1324,15 +1324,15 @@ public Errata publish(User loggedInUser, String advisory, List<String> channelLa
     }
 
     /**
-     * Publishes an existing (unpublished) cloned errata to a set of cloned channels
+     * Publishes an existing cloned errata to a set of cloned channels
      * according to its original erratum
      * @param loggedInUser The current user
      * @param advisory The advisory Name of the errata to publish
      * @param channelLabels List of channels to publish the errata to
      * @throws InvalidChannelRoleException if the user perms are incorrect
      * @return the published errata
      *
-     * @xmlrpc.doc Publishes an existing (unpublished) cloned errata to a set of cloned
+     * @xmlrpc.doc Publishes an existing cloned errata to a set of cloned
```

Comment:
dow we still need this method? shouldn't it be removed entirely?

Reply:
See https://github.com/uyuni-project/uyuni/pull/2296/files#r436646780